### PR TITLE
Update Makefile

### PR DESCRIPTION
**Pull request by GuruPKK**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,12 +2,20 @@ GIT_BRANCH=`git rev-parse --abbrev-ref HEAD`
 USER=`whoami`
 STAGING_URL="https://docs-mongodborg-staging.corp.mongodb.com"
 PRODUCTION_URL="https://docs.mongodb.org/meta"
-STAGING_BUCKET=docs-mongodb-org-staging
+STAGING_BUCKET=docs-mongodb-org-prd-staging
 PRODUCTION_BUCKET=docs-mongodb-org-prod
```
Comment (jdestefano-mongo) [inline Makefile:6]: Shouldn't this be "docs-mongodb-org-prd"?

File: Makefile [hunk 2]
```diff
@@ -2,12 +2,20 @@ GIT_BRANCH=`git rev-parse --abbrev-ref HEAD`
...
 
 # "PROJECT" currently exists to support having multiple projects
 # within one bucket. For the manual it is empty.
 PROJECT=meta
+PREFIX=ruby-driver
+STGPREFIX=ruby-driver
```
Comment (jdestefano-mongo) [inline Makefile:12]: Can we remove lines 11 and 12? They don't appear to be used...

**Review by jdestefano-mongo — CHANGES_REQUESTED** [review]
@GuruPKK requesting changes on this one

File: Makefile (hunk 1 above)
Comment (GuruPKK) [reply Makefile]: updated

File: Makefile (hunk 2 above)
Comment (GuruPKK) [reply Makefile]: removed

**Review by jdestefano-mongo — APPROVED** [review]

**Pull request merged**